utils/swift_build_support/tests/products/test_ninja.py

```diff
@@ -11,6 +11,7 @@
 
 import argparse
 import os
+import os.path
```

Comment:
There should be no need to explicitly import `os.path` if you already import `os`.